togetherly/lib/views/widgets/chore_list.dart

```diff
@@ -49,10 +25,9 @@ class _ChoreListState extends State<ChoreList> {
           style: AppTextStyles.brandAccentLarge,
         ),
         Column(
-          children: <Widget>[
-            for (final chore in chores)
-              ChoreItem(chore: chore),
-          ],
+          children: widget.choreList.map((chore) {
+            return ChoreItem(chore: chore,);
+          }).toList(),
```

Comment:
Why the change here? I feel that the former syntax was more readable.

Reply:
It's more concise and utilizes more of Darts list manipulation methods. And it's more consistent. Using a map with toList() is used more frequently throughout the app than a for loop, including in the service. I only have one remaining for loop, which I'm going to replace with a map method.